thunder/executors/cudnnex.py

```diff
@@ -282,7 +282,8 @@ def _cudnn_sdpa_fwd_impl(
     value = _sdpa_enforce_input_tensor_contiguity(value)
 
     if attn_mask is not None:
-        attn_mask = attn_mask.view((1,) * (query.ndim - attn_mask.ndim), *attn_mask.shape)
+        if query.ndim > attn_mask.ndim:
+            attn_mask = attn_mask.view((1,) * (query.ndim - attn_mask.ndim), *attn_mask.shape)
```

Comment:
Should this be 

`attn_mask.view(*((1,) * (query.ndim - attn_mask.ndim)), *attn_mask.shape)`

Eg.
```python
>>> import torch
>>> query = torch.randn(3, 3)
>>> attn_mask = torch.randn(3)
>>> (1,) * (query.ndim - attn_mask.ndim), *attn_mask.shape
((1,), 3)
>>> *((1,) * (query.ndim - attn_mask.ndim)), *attn_mask.shape
(1, 3)
>>> attn_mask = torch.randn(3, 3)
>>> (1,) * (query.ndim - attn_mask.ndim), *attn_mask.shape
((), 3, 3)
>>> *((1,) * (query.ndim - attn_mask.ndim)), *attn_mask.shape
(3, 3)
```